typos and minor edits (#326)
Benjamin-Lee/deep-rules: content/13.conclusion.md

## Conclusion {#conclusion}

> Collectively, our manuscript is focused on the promotion of practical tips, cutting-edge insights, and evolving professional standards that can help to advance the efficient and optimal application of deep learning across a wide range of scientific research environments.

Reviewer: ```suggestion
Collectively, our manuscript is focused on the promotion of practical tips distilled from cutting-edge insights, and evolving professional standards that can help to advance the efficient and optimal application of deep learning across a wide range of scientific research environments.
```

Author: @Benjamin-Lee I like your version but I made a tiny change - what do you think?

> Specifically, we have tried to increase the accessibility of deep learning techniques within the life sciences so that these cutting edge techniques can generate innovative biomedical insights, tools, technologies, services, and products.

Reviewer: ```suggestion
Specifically, we have tried to increase the accessibility of deep learning techniques within the life sciences so that these these methods can generate innovative biomedical insights, tools, technologies, services, and products.
```

> A number of our points (see Tips 7, 8, 9, and 10) are deeply motivated by the need to safeguard against key risks, which include the introduction/perpetuation of bias, the overinterpretation/misinterpretation of models, poor generalizability, and the potential for harm unto others.

Reviewer: ```suggestion
A number of our tis (see Tips 7, 8, 9, and 10) are deeply motivated by the need to safeguard against key risks, which include the introduction and perpetuation of bias, over- or mis-interpretation of models, poor generalizability, and the potential for harm unto others.
```

> Despite these concerns, deep learning techniques have the potential to add value across a diverse array of research and healthcare contexts if they are leveraged in ethical and responsible ways.

Reviewer: ```suggestion
However, despite these concerns, deep learning techniques have the potential to add value across a diverse array of research and healthcare contexts if they are leveraged in ethical and responsible ways.
```

## Conclusion {#conclusion}

Collectively, our manuscript is focused on the promotion of practical tips, cutting-edge insights, and evolving professional standards that can help to advance the efficient and optimal application of deep learning across a wide range of scientific research environments.
A number of our points (see Tips 7, 8, 9, and 10) are deeply motivated by the need to safeguard against key risks.
These include the introduction/perpetuation of bias, the overinterpretation/misinterpretation of models, poor generalizability, and the potential for harm unto others, which can have a mix of ethical, legal, and social implications.

> Despite these concerns, deep learning techniques have the potential to add value across a diverse array of research and healthcare contexts if they are leveraged in ethical and responsible ways. After all, these techniques have already shown a remarkable ability to meet or exceed the performance of human effort and older algorithms across numerous fields and subdisciplines.

Reviewer 2: ```suggestion
Despite these concerns, deep learning techniques have the potential to add value across a diverse array of research and healthcare contexts if they are leveraged in ethical and responsible ways.
After all, these techniques have already shown a remarkable ability to meet or exceed the performance of human effort and older algorithms across numerous fields and subdisciplines.
```

Beyond merely achieving good predictive performance in certain tasks, deep learning also has the potential to uncover hidden biological and clinical patterns of potentially high impact, which could fundamentally drive research discoveries and the delivery of new products to market.
On the whole, deep learning must be approached with genuine thoughtfulness, caution, and responsibility if it is to reach its full potential.

> Through the tips and recommendations provided within this manuscript, we have tried to increase the accessibility of deep learning techniques within the life sciences.  This is of critical importance given that published findings, models, and datasets will be leveraged to yield innovative tools, services, and products across societal spheres.

Reviewer 2: ```suggestion
Through the tips and recommendations provided within this manuscript, we have tried to increase the accessibility of deep learning techniques within the life sciences.
This is of critical importance given that published findings, models, and datasets will be leveraged to yield innovative tools, services, and products across societal spheres.
```

> We also hope to promote the development of a prudent and vigilant community of computational practitioners, experimental biologists, and clinical scientists that pauses to think, discuss, and carefully plan before excitedly stitching together lines of code that improperly apply deep (and tradition) learning techniques.

Reviewer 2: ```suggestion
We also hope to promote the development of a prudent and vigilant community of computational practitioners, experimental biologists, and clinical scientists that pauses to think, discuss, and carefully plan before excitedly stitching together lines of code that improperly apply deep (and traditional) learning techniques.
```

> We also hope to promote the development of a prudent and vigilant community of computational practitioners, experimental biologists, and clinical scientists that pauses to think, discuss, and carefully plan before excitedly stitching together lines of code that improperly apply deep (and tradition) learning techniques.

Reviewer: Can we make this sentence more positive? I do want to end on an uplifting note.

Author: I tried to edit it to do that - what do you think?

> After all, the initial discernment of the potential scientific and societal consequences of one's work is critical to the advancement of accountability, beneficence, and quality in science. By following the tips we have outlined and the lessons they underpin, such a community can promote deep learning research that is of high quality, interpretability, and reproducibility.

Reviewer 2: These closing sentences are pretty words.  Can we close with a shorter, punchier take home message?

Author: I agree - I previously tried to condense the conclusion in this most recent edit, as it seemed to repeat a few core ideas, but I just took another swing at condensing it further.
